src/main/java/uk/gov/justice/digital/job/DomainRefreshJob.java

```diff
@@ -55,8 +58,15 @@ public static void main(String[] args) {
 
     @Override
     public void run() {
-        DomainRefreshService domainRefreshService = refresh();
-        domainRefreshService.run(domainTableName, domainId, domainOperation);
+        DomainService domainRefreshService = refresh();
+        try {
+            domainRefreshService.run(domainTableName, domainId, domainOperation);
+        } catch (PatternSyntaxException e) {
+            final StringWriter sw = new StringWriter();
```

Comment:
This should just be a single call to the logger at error. No need to send anything to STDOUT.